ios/tutorials/tutorial4-completed/TicTacToe/AppStart/AppDelegate.swift

```diff
@@ -39,7 +39,7 @@ public class AppDelegate: UIResponder, UIApplicationDelegate {
         let result = RootBuilder(dependency: AppComponent()).build()
         launchRouter = result.launchRouter
         urlHandler = result.urlHandler
-        launchRouter?.launchFromWindow(window)
+        launchRouter.launch(from: window)
```

Comment:
@jarinosuke looks like the `?` was removed.

Reply:
@sbarow  I'm not sure why this used to be declared as optional, but it looks OK as it is.
![screen shot 2017-11-28 at 18 37 35](https://user-images.githubusercontent.com/186187/33312438-3ca289d2-d46b-11e7-803d-32447ec4294e.png)